compiler/noirc_frontend/src/hir/comptime/interpreter/builtin.rs

```diff
@@ -48,6 +48,10 @@ impl<'local, 'context> Interpreter<'local, 'context> {
             "array_len" => array_len(interner, arguments, location),
             "as_slice" => as_slice(interner, arguments, location),
             "expr_as_function_call" => expr_as_function_call(arguments, return_type, location),
+            "expr_as_if" => expr_as_if(arguments, return_type, location),
+            "expr_as_index" => expr_as_index(arguments, return_type, location),
+            "expr_as_parenthesized" => expr_as_parenthesized(arguments, return_type, location),
```

Comment:
I don't think we should expose Ast::Parenthesized, I see it more of an internal detail to preserve formatting.
Instead we should probably have a check for it in `expr_as` to automatically recur and match on the expression inside of it instead.

Reply:
Ah, I also thought about this and I wasn't sure. I'll do it 👍